pkgs/development/python-modules/sphinx-jsonschema/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  docutils,
+  requests,
+  jsonpointer,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-jsonschema";
+  version = "1.19.1";
+
```

Comment:
Missing format = "setuptools"

Reply:
Hey thanks for the review! Just two quick questions: Isn't format = "setuptools" the default anyway, so what difference does it make adding it?

Reply:
It is the default, yes. But I think it is good to have it anyways, since it shows the packager thought about it.

Reply:
makes sense, added it now